classes/class6/README.md

```diff
@@ -1 +1,96 @@
+
+>**URL Endpoint:** https://api.twitter.com/1.1
+
+>**Data Format**: JSON
+
+>**Account Creation:** Automatic key assignment
+
+>**Auth Scheme:** OAuth 1.0a
+
+>**Documentation:** https://dev.twitter.com/rest/public
+
+###Last.fm
+Get information on music artist, albums, and individual tracks. Get listening trend data, even venue and event info. You can also use this API to record users' track listens (called scrobbling) to Last.fm. **Examples:** [Tastebuds Music-based Dating](http://tastebuds.fm/), [Last.fm Extra Stats](http://www.last.fm/user/C26000/journal/2006/07/30/383m_last.fm_extra_stats), [Spotibot Playlist Generator](http://www.spotibot.com/)
+
+>**URL Endpoint:** http://ws.audioscrobbler.com/2.0/
+
+>**Data Format**: XML/JSON if requested
+
+>**Account Creation:** Apply for key
+
+>**Auth Scheme:** Proprietary
+
+>**Documentation:** http://www.last.fm/api
+
+###Dropbox
+Access all your files stored online! Put them up, take them down, share them with others, read metadata and revision history. The possibilities really are endless on this one. **Examples:** [Filepicker upload plugin](https://www.filepicker.io/), [Gimmebar Browse Plugin](https://gimmebar.com/), [Droptunes Music Streaming](http://droptun.es/)
+
+>**URL Endpoint:** https://api.dropbox.com/1
+
+>**Data Format**: JSON & Various File Formats
+
+>**Account Creation:** Automatic key assignment
+
+>**Auth Scheme:** OAuth 1.0
+
+>**Documentation:** https://www.dropbox.com/developers
+
+###Netflix
+This one should not be on the list of APIs. Sadly, they closed their public API last November. Let's all take a moment to boo Netflix. Booooooooo.
+
+##Accessing Public APIs with OAuth
+You might have noticed that some of the above APIs are listed as having an OAuth auth scheme, but you probably have no idea what that means. Sometimes, a 3rd party application might want access to private user data, but you can't reasonably expect web service providers to give that information out freely - we need a way for service providers' users to authorize 3d party applications to access their data. OAuth lets us do just that. **Note:** We're going to assume that throughout this class you will only be consuming APIs which require OAuth and not creating them.
+
+###A brief overview of OAuth
+OAuth plays two roles in accessing data - *authentication* and *authorization*. 
+
+Authentication is handled by assigning two unique values to each application that wants to use the service. These values are known as a **key** and a **secret**. The key serves as a public identifier for the application, and the secret is used to reversibly encrypt information about an OAuth API request. If the service provider is able to correctly decrypt this information with the secret it has assigned the application, it considers this verification that the requester is who it says it is. This method of authentication is known as signing.
```

Comment:
"If the service provider is able to correctly decrypt *this information* with the secret it has assigned the application"  Can you make what 'this information' is more clear?
assigned the -> assigned to the
Maybe also mention that most APIs will require you to register your app or otherwise send a request to get a key and secret. (It's implied, but could be more clear in the first sentence)